### ISSUE TITLE
As a developer submitting my dapp, I want to be able to add my own blockchain, storage, etc while submitting, so I'm not limited to an arbitrary list of protocols

### ISSUE THREAD

**Issue by jeffdomke** [blockstack/app.co#4]

**Comment by hstove** [comment]
Yeah, I'd probably just implement it like it's an "other" option, with a user-specified issue. We wouldn't want to display these categories as their own pages, like you said, and we'd probably want to just watch what people are submitting, and then add them as official categories. When I say category, I mean blockchain/storage/auth, not actual categories like "Social Networking"

**Comment by markmhx** [comment]
I like that approach – just let them fill in a string if they pick "Other" and we can see what data we get and map it out later if we see trends.

**Comment by markmhx** [comment]
@hstove I spoke with @shea256 about this issue today since he raised it as an example of work that we could have done better prioritizing already.

Could you prioritize work on it for immediately after the completion of this project? https://github.com/blockstack/app.co/projects/6

**Comment by hstove** [comment]
Yes, we can prioritize it. However, my gut feeling is that it's not a high priority, relative to other things like improving app page content. My main reasoning is that we don't even know what other platforms people need for support. We already support more platforms than other dapp stores, and I've never seen a dapp platform that we don't support.

The counterargument is that we don't get submissions from other platforms, because people think it's not supported.

**Comment by jeffdomke** [comment]
Same reaction as hank. Not getting this. I agree we should be encouraging a diversity of protocols but I think we should do this in a technically, and product complexity, cheap way. 

- What alternatives were considered? A mailto link that says "Don't see your protocol? We'll add it." 
- We have a handful of these that were added that are sitting empty. Why do we want to add others that will have few/1 apps?

**Comment by markmhx** [comment]
I think it’ll be pretty cheap to add an other field and just start capturing string values, wouldn’t it? We could then decide later how to use them, especially if we spot trends in entries and want to spin up new platforms, etc around them.

**Comment by markmhx** [comment]
Also, I think @shea256’s main rationale here is that we want to avoid creating an exclusive experience for any developer while they’re submitting an app from across the ecosystem, even if we don’t leverage their custom data specifically.

**Comment by markmhx** [comment]
I think the very lean solution here (aimed at simply unblocking submitters who can’t currently find their platform, etc) would be to implement the first two items in @hstove’s list above. 

As such, we’d start collecting the data as strings but do nothing with it (at least yet). And we can punt on auto-suggestions until we find that too many devs are submitting “other” values for values we actually support but they somehow missed.

**Comment by markmhx** [comment]
Per our conversation today during the stand-up, we've decided to move forward with the "first two items" approach above.

@hstove @jeffdomke please let me know if you're remembering the convo differently, but otherwise I think we're good to go with a quick implementation of this.
